graph/schema.resolvers.go

```diff
@@ -41,7 +41,17 @@ func (r *queryResolver) Messages(ctx context.Context) ([]*model.Message, error)
 	return resolver.Messages(ctx)
 }
 
+// SearchSubscription is the resolver for the search field.
+func (r *subscriptionResolver) Search(ctx context.Context, input []*model.SearchInput) (<-chan []*resolver.SearchResult, error) {
+	klog.Infoln("Received search query subscription")
```

Comment:
Increase verbosity to avoid flooding the log.
```suggestion
	klog.V(3).Info("Received search query subscription")
```